reporting-cli/docs/user/Reference Manual.md

```diff
@@ -0,0 +1,131 @@
+│   └── .env
+```
+
+- Each line should have format NAME=VALUE
+- Lines starting with # are considered as comments.
+- There is no special handling of quotation marks.
+
+NOTE: Values from the environment file has higher priority than command line argument. For example, if you add filename as *test* in *.env* file and also add `--filename report` command option, the downloded report's name will be *test*.
+
+### Example
+
+Sample command for downloading a dashboard report with basic authentication in png format
+```
+reporting --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --format png --auth basic --credentials admin:admin
+```
+Report will be downloaded in the current directory.
+
+### Sending an Email with report attachment using Amazon SES
+
+Prerequisites:
+- The sender's email address must be verified on [Amazon SES](https://aws.amazon.com/ses/).
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) is required to interact with Amazon SES. 
+- [Configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) basic settings used by AWS CLI.
+-  SES transport requires ses:SendRawEmail role.
+```
+{
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "ses:SendRawEmail",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+Sample command to send email with report as an attachment:
+```
+reporting --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport SES --from <sender_email_id> --to <recipient_email_id>
+```
+This example uses default values for all other options.
+
+You can also set *FROM*, *TO*, *TRANSPORT* in .env file and use following command. 
+```
+reporting --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d
+```
+
+To modify the body of your email, you can simply edit *index.hbs* file.
+
+
+### Sending an Email with report attachment using SMTP
+
+For sending email using SMTP transport, the options **SMTP_HOST**, **SMTP_PORT**, **SMTP_USER**, **SMTP_PASSWORD**, **SMTP_SECURE** need to be set in .env file.
+
+Once above options are set in .env file, you can send the email using below sample command.
+```
+reporting --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport SMTP --from <sender_email_id> --to <recipient_email_id>
+```
+
+You can choose to set options using *.env* file or the command line argument values in any combination. Make sure to specify all required values to avoid getting errors.
+
+To modify the body of your email, you can simply edit *index.hbs* file.
+
+## Limitations
+- If a URL contains `!`, history expansion needs to be disable temporarity. 
+
+  bash: `set +H`
+
+  zsh: `setopt nobanghist`
+
+
+  Alternate option would be adding URL value in envirnoment variable as URL="<url-with-!>" 
+
+- All command option currently accept only lower-case letters.
```

Comment:
why is this? in the example transport option `SMTP` is not lowercase, if you meant the actual option flag text i think that's expected?

Reply:
I updated the examples but the reason is the choices are added as `.choices(['ses', 'smtp'])`.  Not sure how to add ignore-case. Need to check.